pkg/gds/store/trtl/trtl.go

```diff
@@ -0,0 +1,92 @@
+package trtl
+
+import (
+	"errors"
+
+	"github.com/rs/zerolog/log"
+	"github.com/trisacrypto/directory/pkg/gds/client"
+	"github.com/trisacrypto/directory/pkg/gds/models/v1"
+	"github.com/trisacrypto/directory/pkg/gds/store/iterator"
+	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
+	gds "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+	"google.golang.org/grpc"
+)
+
+// Open a connection to the Trtl database.
+func Open(profile *client.TrtlProfile) (store *Store, err error) {
+	store = &Store{}
+	if store.conn, err = profile.Connect(); err != nil {
+		return nil, err
+	}
+	store.client = pb.NewTrtlClient(store.conn)
+	return &Store{}, nil
+}
```

Comment:
Do we plan on integrating this into the generic `store.Store.Open`? It seems like it might be a bit awkward, because that method is expecting to deal with embedded databases and we need to somehow pass in the mTLS parameters.

Reply:
As discussed, we'll go with making the `store.Store.Open` a bit more explicit with the database config.